packages/libcloudproviders.rb

```diff
@@ -1,9 +1,9 @@
 # Adapted from Arch Linux libcloudproviders PKGBUILD at:
 # https://github.com/archlinux/svntogit-packages/raw/packages/libcloudproviders/trunk/PKGBUILD
 
-require 'package'
+require 'buildsystems/meson'
 
-class Libcloudproviders < Package
+class Libcloudproviders < Meson
   description 'DBus API that allows cloud storage sync clients to expose their services'
   homepage 'https://gitlab.gnome.org/World/libcloudproviders'
   version '0.3.1'
```

Comment:
Update to [0.3.5](https://gitlab.gnome.org/World/libcloudproviders/-/tags/0.3.5)?